functions/v2/audit_log/main.py

```diff
@@ -14,25 +14,29 @@
 
 
 # [START functions_log_cloudevent]
+import functions_framework
+
+
 # CloudEvent function to be triggered by an Eventarc Cloud Audit Logging trigger
 # Note: this is NOT designed for second-party (Cloud Audit Logs -> Pub/Sub) triggers!
-def hello_auditlog(cloudevent):
+@functions_framework.cloud_event
+def hello_auditlog(cloud_event):
     # Print out the CloudEvent's (required) `type` property
     # See https://github.com/cloudevents/spec/blob/v1.0.1/spec.md#type
-    print(f"Event type: {cloudevent['type']}")
+    print(f"Event type: {cloud_event['type']}")
```

Comment:
Nit: is there a reason why we shouldn't use `cloudevent` here?

(I think we still do that in other languages/samples.)

Reply:
This is due to the discussion of "Cloud Event" being 2 words so we should use snake case in code for consistency everywhere: https://github.com/GoogleCloudPlatform/functions-framework-python/issues/166